From be8e536e068adaced0ab5aa8d5824c0d16ed3ba8 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 26 Dec 2018 11:06:31 +0200
Subject: [PATCH] [SW-1886] pll: reduce Fvco to 1200MHz

We observe a Synchronous Aborts in about 0.5-2% of the reboots.
In our tests this has been pinned down to a PLL problem.
Statistically it was found that reducing the Fvco from 2400 MHz to
1200MHz (regardless the theory) improves the situation dramatically.
While investigation continues, reduce the Fvco of the CPU PLL to
1200MHz.

Change-Id: I7a4cdf72fb63a619539236eb90da59acd38b38fc
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/pll.c b/pll.c
index 6387f7d..dfc8fff 100644
--- a/pll.c
+++ b/pll.c
@@ -47,7 +47,7 @@
 
 static struct hl_pll_cfg goya_pll_cfg[] = {
 	{.name="CPU", .base=(void*)mmCPU_PLL_BASE,
-	 .nr=0, .nf=47, .od=1, .nb=23, .div_factor={0,23,1,-1}},
+	 .nr=0, .nf=23, .od=0, .nb=11, .div_factor={0,23,1,-1}},
 	{.name="IC", .base=(void*)mmIC_PLL_BASE,
 	 .nr=0, .nf=63, .od=15, .nb=31, .div_factor={0,1,1,2}},
 	{.name="MME", .base=(void*)mmPSOC_MME_PLL_BASE,
